.circleci/config.yml

```diff
@@ -8,9 +8,6 @@ jobs:
       - run:
           name: install
           command: npm install
-      - run:
```

Comment:
?

Reply:
It's being a bitch. 

1. Running it on a branch that is going to be squashed and merge is silly. 
1. It doesn't actually prevent us from putting the wrong title when we finally merge to master.